docker-compose.yml

```diff
@@ -1,53 +1,79 @@
 services:
   patron_c2_server:
-    image: patron_c2_server
-    build: server
+    image: patron-server
+    build:
+      context: .
+      dockerfile: Dockerfile.server
+    environment:
+      DB_HOST: patron_c2_postgres
+      DB_PORT: ${DB_PORT}
+      DB_USER: ${DB_USER}
+      DB_PASS: ${DB_PASS}
+      DB_NAME: ${DB_NAME}
+      C2SERVER_IP: patron_c2_server
+      C2SERVER_PORT: ${C2SERVER_PORT}
     restart: always
-    command: sh -c "go run server/server.go"
-    working_dir: /app
     volumes:
-      - ./:/app
+      - ./logs:/app/logs
+      - ./certs:/app/certs
     networks:
-      internal_c2_network:
-        ipv4_address: "${DOCKER_INTERNAL}.5"
+      - internal_c2_network

```

Comment:
irc the last octet was defined on purpose. Did you test this out?

Reply:
yes, it is using the internal Docker DNS. Tested on 3 separate boxes and all of them functioned fully